<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdtServer</class>
 <widget class="QMainWindow" name="UpdtServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdtServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_Label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
